packages/console/src/pages/TenantSettings/tabs/TenantDomainSettings/index.tsx

```diff
@@ -4,16 +4,22 @@ import useSWR from 'swr';
 
 import FormCard from '@/components/FormCard';
 import FormField from '@/components/FormField';
-import { type RequestError } from '@/hooks/use-api';
+import useApi, { type RequestError } from '@/hooks/use-api';
+import useSwrFetcher from '@/hooks/use-swr-fetcher';
 
 import AddDomainForm from './components/AddDomainForm';
 import CustomDomain from './components/CustomDomain';
 import DefaultDomain from './components/DefaultDomain';
 import * as styles from './index.module.scss';
 
 function TenantDomainSettings() {
-  // Todo: @xiaoyijun setup the auto refresh interval for the domains when implementing the active domain process.
-  const { data, error, mutate } = useSWR<Domain[], RequestError>('api/domains');
+  const api = useApi();
+  const fetcher = useSwrFetcher<Domain[]>(api);
```

Comment:
Why not using the default fetcher?

Reply:
The useSwrFetcher is the default fetcher.
Since we need to specify the `refreshInterval`, we need use `useSWR` in this way.